javascript-light-bulb/script.js

```diff
@@ -1,29 +1,32 @@
-var off = 0;
+var off = false;
 var $body = document.querySelector('.background-color-off');
 var $lightBulb = document.querySelector('.light-bulb-off');
 
 $lightBulb.addEventListener('click', function () {
-  off++;
-  if (off % 2 === 0) {
+  if (off === false) {
```

Comment:
I would reverse your conditional blocks and make the conditional `if(off)`, it reads much cleaner 🤓